### ISSUE TITLE
Syntax error

### ISSUE THREAD

**Issue by mcmire** [Rich-Harris/sorcery#9]
When using sorcery I'm getting this error:

```
~/code/personal/paper-js-experiments/node_modules/gobble/node_modules/sorcery/dist/sorcery.js:566
		var content = this.node.content.replace(SOURCEMAP_COMMENT, '') + `\n//# sour
		                                                                 ^
Unexpected token ILLEGAL
```

It looks like the string literal is throwing off Node. [String literals were added in v8 3.31](https://github.com/joyent/node/wiki/ES6-%28a.k.a.-Harmony%29-Features-Implemented-in-V8-and-Available-in-Node#string-literals), but Node 0.12.2 ships with 3.28.

What was the reasoning behind using a string literal here, and how are you getting this to work?

**Comment by Rich-Harris** [comment]
Whoops! Sorcery is written in ES6 and transpiled to ES5 with babel - the [babel transformer whitelist](https://github.com/Rich-Harris/sorcery/blob/master/gobblefile.js#L7-L17) was missing the template literal transformer, which meant the template literals were passed through untouched.

I didn't notice it because I'm running [io.js](https://iojs.org/en/index.html) instead of node. Have [hooked this repo up to Travis CI](https://travis-ci.org/Rich-Harris/sorcery) to stop anything similar happening in future.

It's fixed in 0.5.1. Thanks for raising it.

**Comment by mcmire** [comment]
Awesome, thanks for the quick response/fix!
